mlos_bench/mlos_bench/storage/sql/schema.py

```diff
@@ -46,6 +46,9 @@ class DbSchema:
     A class to define and create the DB schema.
     """
 
+    _LEN_ID = 255
+    _LEN_VALUE = 255
```

Comment:
Additionally, does this need to be so short even for non-indexed value columns?  Or do we need to change the type on some of them to `TEXT`?

Reply:
255 is what we have now for both IDs and values. Let's have 512/1024 instead and move on. We can always review it later. Right now I just want to unblock @eujing